policy/modules/system/systemd.if

```diff
@@ -1046,6 +1046,7 @@ interface(`systemd_manage_passwd_run',`
 	manage_files_pattern($1, systemd_passwd_var_run_t, systemd_passwd_var_run_t)
 	manage_sock_files_pattern($1, systemd_passwd_var_run_t, systemd_passwd_var_run_t)
 	manage_fifo_files_pattern($1, systemd_passwd_var_run_t, systemd_passwd_var_run_t)
+	watch_dirs_pattern($1, systemd_passwd_var_run_t, systemd_passwd_var_run_t)
```

Comment:
Just some additional info about this event:

The watch is for the ask-password path units [1]. systemd watches all path unit objects  [2]. Might be a better way to address this automatically for all potential path units by associatiing path unit specific rules and types with a path unit object type attribute,

[1] ls /usr/lib/systemd/system | grep .path$
[2] man systemd.path

Reply:
I actually was reminded about this myself just this morning:

https://defensec.nl/gitweb/dssp2.git/commitdiff/883a71a2e383ba0b3c1c71082e86fe6ca9b0dfcf

Reply:
Oh, right, I see... I'll try to restructure that part in a better way. Thanks for the review!